tools/tertiary-analysis/pyscenic/pyscenic_aucell.xml

```diff
@@ -0,0 +1,108 @@
+        <param name="signatures_fname" type="data" format="csv,tabular" label="Gene Signatures/Regulons File" help="The file that contains the gene signatures (usually the precomputed regulons). Currently only csv/tsv supported, could be extended."/>
+        <param name="transpose" type="boolean" label="Transpose Expression Matrix" help="Transpose the expression matrix if supplied as csv (rows=genes x columns=cells)." optional="true"/>
+        <param name="weights" type="boolean" label="Use Weights for Recovery Analysis" help="Use weights associated with genes in recovery analysis. Relevant when gene signatures are supplied as json format." optional="true"/>
+        <param name="seed" type="integer" label="Seed for Ranking" help="Seed for the expression matrix ranking step. The default is to use a random seed." optional="true"/>
+        <param name="rank_threshold" type="integer" label="Rank Threshold" help="The rank threshold used for deriving the target genes of an enriched motif (default: 5000)." optional="true"/>
+        <param name="auc_threshold" type="float" label="AUC Threshold" help="The threshold used for calculating the AUC of a feature as fraction of ranked genes (default: 0.05)." optional="true"/>
+        <param name="nes_threshold" type="float" label="NES Threshold" help="The Normalized Enrichment Score (NES) threshold for finding enriched features (default: 3.0)." optional="true"/>
+        <param name="cell_id_attribute" type="text" label="Cell ID Attribute" help="The name of the column attribute that specifies the identifiers of the cells in the loom file." optional="true"/>
+        <param name="gene_attribute" type="text" label="Gene Attribute" help="The name of the row attribute that specifies the gene symbols in the loom file." optional="true"/>
+        <param name="sparse" type="boolean" label="Sparse Matrix" truevalue="--sparse" falsevalue="" help="If set, load the expression data as a sparse matrix. Currently applies to the grn inference step only."/>
+    </inputs>
+    <outputs>
+        <data name="output" format="tsv" label="${tool.name} on ${on_string}: AUCell scores for regulons or gene sets."/>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="1">
+            <param name="expression_mtx_fname" value="expr_mat.loom"/>
+            <param name="signatures_fname" value="regulons.tsv"/>
+            <output name="output">
+                <assert_contents>
+                    <has_n_lines n="101"/>
+                    <has_text text="CEBPB"/>
+                </assert_contents>
+            </output>
+        </test>
+    </tests>
+    <help>
+        <![CDATA[
+          Run PySCENIC aucell command to analyze single-cell gene expression data.
```

Comment:
This help is not very useful for a user, it just repeats the parameter help. Better would be an explanation of the tool and its inputs/outputs

Reply:
I will leave it to contributors interested in doing tutorials to improve on this. My use of this is mostly automatic.